packages/installer/src/ethClient/apiUrl.ts

```diff
@@ -27,27 +27,28 @@ export function getEthExecClientApiUrl(dnpName: string, port = 8545): string {
  * @param dnpName
  */
 export async function getEthConsClientApiUrl(dnpName: string): Promise<string> {
-  let port = 3500;
-  let domain = "";
+  const defaultPort = 3500;
+  const defaultServiceName = "beacon-chain";
+
   const dnp = await listPackageNoThrow({ dnpName });
-  if (dnp && typeof dnp.chain === "object" && dnp.chain.portNumber && dnp.chain.serviceName) {
-    port = dnp.chain.portNumber;
-    domain = buildNetworkAlias({
-      dnpName: dnpName,
-      serviceName: dnp.chain.serviceName,
-      isMainOrMonoservice: false
-    });
-  } else {
-    // Lighthouse, Teku and Prysm use 3500
-    // Nimbus uses 4500 because it is a monoservice and the validator API is using that port
-    if (dnpName.includes("nimbus")) {
```

Comment:
All production Nimbus clients have their driver defined like this:

```
  "chain": {
    "driver": "ethereum-beacon-chain",
    "serviceName": "beacon-validator",
    "portNumber": 4500
  },
  ```
  
  So it is not necessary to consider this condition. Also, once nimbus package is published from https://github.com/dappnode/DAppNodePackage-nimbus-generic , it will have 2 services that will behave the same as the rest of the clients